lib/bot.js

```diff
@@ -98,6 +100,24 @@ class Bot {
       this.sendToDiscord(author, to, `*${text}*`);
     });
 
+    this.ircClient.on('join', (channel, nick) => {
+      if (this.disableJoinPartMsgs) return;
+      if (nick === this.nickname && !this.announceSelfJoin) return;
```

Comment:
I see. Could keep it as a non-documented option, but as you're saying I think it might make more sense to print something like `The bot has joined the IRC channel.` then. Not sure how you'd be able to announce if the bridge is down though, as I think most issues with crashes have been on the Discord side (i.e. timing out and not reconnecting), but I guess you could at least print if the IRC connection goes offline. Seems a bit of an edge case though.